core/src/com/unciv/models/ruleset/unique/UniqueTriggerActivation.kt

```diff
@@ -295,6 +295,23 @@ object UniqueTriggerActivation {
                     true
                 }
             }
+            UniqueType.OneTimeRemovePolicy -> {
+                val policyName = unique.params[0]
+                if (!civInfo.policies.isAdopted(policyName)) return null
+                val policy = civInfo.gameInfo.ruleset.policies[policyName] ?: return null
+
+                return {
+                    civInfo.policies.removePolicy(policy)
```

Comment:
Personal request: Can this make use of filters? There is actually a good usecase here for filters with removing whole branches if a user wants to switch branches (such as a mod trying too hard to emulate BNW)

Reply:
If I understood your words correctly, you are suggesting that we could implement a policyFilter that accepts either a policy name or a policy branch name as a parameter ?

That's a good idea, I would consider implement it in another PR

Reply:
Well, that, an the usual option of filtering uniques. Then this could just remove all policies that fit the filter. Then only real limitation would be that the filter for the branch itself can't be the name of the branch because that refers to the opener policy